pir/cpp/payload.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package pir;
+
+message Payload {
+  repeated bytes query = 1;
+}
+
+message SessionPayload {
```

Comment:
What do you mean by SessionPayload? Is this a query for a session value? It looks like a query to me. Perhaps this should be PIRQuery? Please add a comment description as well.

Reply:
Payload is used for both query and response.

The session name comes from optimizing the GaloisKey roundtrips.